AppCreationScripts/apps.json

```diff
@@ -0,0 +1,35 @@
+{
+	"Sample": {
+		"Title": "Calling Microsoft Graph from a macOS app",
+		"Level": 400,
+		"Client": "macOS"
+	},
+	"AppRegistrations": [{
+		"x-ms-id": "active-directory-macOS-swift",
+		"x-ms-name": "macOS-swift",
+		"x-ms-version": "2.0",
+		"replyUrlsWithType": [{
+			"url": "msal{AppId}://auth",
+			"type": "InstalledClient"
+		}],
+		"requiredResourceAccess": [{
+			"x-ms-resourceAppName": "Microsoft Graph",
+			"resourceAppId": "00000003-0000-0000-c000-000000000000",
+			"resourceAccess": [{
+				"id": "e1fe6dd8-ba31-4d61-89e7-88639da4683d",
+				"type": "Scope",
+				"x-ms-name": "user.read"
+			}]
+		}],
+		"codeConfigurations": [{
+				"settingFile": "/MSALMacOS/ViewController.swift",
+				"replaceTokens": {
```

Comment:
how does Redirect URI and/or bundleId get replaced?

Reply:
Based on my knowledge, bundle id is only required on iOS app, not for mac. 
Redirect URL is nil in the sample app and we are not declaring it in the file 

let me know me if I am wrong with it

Reply:
RedirectUri is also required for macOS. It should be in the format of msauth.<bundleId>://auth, so some replacement will be needed. If we don't do it yet, or do something else, it needs to be fixed.

Reply:
That reminds me of the mistakes I made when using this sample app against my portal app that I built in my first week. Need to configure the redirect url otherwise it won’t work. Will make the fix on both platforms